certbot/storage.py

```diff
@@ -392,6 +392,26 @@ def __init__(self, config_filename, cli_config, update_symlinks=False):
         self._check_symlinks()
 
     @property
+    def key_path(self):
```

Comment:
I hate that they're not named consistently, but you're right, this is dead code and should be removed.